src/main/java/com/qunite/api/web/controller/QueueController.java

```diff
@@ -94,7 +97,10 @@ public ResponseEntity<Void> enrollMember(@PathVariable Long id,
   @DeleteMapping("/{id}/members/{member-id}")
   @Operation(summary = "Delete member from queue by id", responses = {
       @ApiResponse(responseCode = "204"),
-      @ApiResponse(responseCode = "404", content = @Content())
+      @ApiResponse(responseCode = "403", description = "User is not a creator or manager",
+          content = @Content(schema = @Schema(implementation = ExceptionResponse.class))),
+      @ApiResponse(responseCode = "404", description = "Could not find member in queue",
```

Comment:
I agree with you, but there is a trick. 
I have written this docs depending on the code we have in services. There is a listing of the deleteMember method, which is invoked at this mapping:
![image](https://github.com/q-unite/qunite-api/assets/93136711/e3d9b83a-99cc-4c64-82e7-602d7ffc8769)
In the line 98 boolean isUserQueueCreatorOrManagerByCredentials is invoked. It gives us false if there is no queue present:
![image](https://github.com/q-unite/qunite-api/assets/93136711/75524102-1d53-46f8-8888-d94da958a97e)
And so ForbiddenException will be thrown and it will lead us to 403 instead of 404.
That is why, in our case, existing description is the most accurate.